sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosResponseDiagnostics.java

```diff
@@ -31,7 +38,12 @@ CosmosResponseDiagnostics clientSideRequestStatistics(ClientSideRequestStatistic
      */
     @Override
     public String toString() {
-        return this.clientSideRequestStatistics.toString();
+        try {
+            return objectMapper.writeValueAsString(this.clientSideRequestStatistics);
+        }catch (JsonProcessingException e) {
+            logger.error("Error while parsing diagnostics " + e.getOriginalMessage());
```

Comment:
usually errors are logged with the following pattern
```
logger.error("Error while parsing diagnostics", e)
```
any reason we are doing something else?

Reply:
no specific reason , just trying to avoid priting whole error stack . I will move to logger.error("Error while parsing diagnostics", e) in next iteration